rpm/libhybris.spec

```diff
@@ -331,8 +330,6 @@ rm %{buildroot}/%{_libdir}/*.la %{buildroot}/%{_libdir}/libhybris/*.la
 %{_libdir}/libandroid-properties.so.*
 %{_bindir}/getprop
 %{_bindir}/setprop
-%{_libdir}/libhybris/linker/mm.la
```

Comment:
Once upstream has set, we'll be packaging all available linkers. It will be shown in the new PR, closing this one.